.env

```diff
@@ -78,3 +78,6 @@ BLACKFIRE_SERVER_TOKEN=<your_backfire_server_token>
 # MESSENGER_TRANSPORT_DSN=amqp://guest:guest@localhost:5672/%2f/messages
 # MESSENGER_TRANSPORT_DSN=redis://localhost:6379/messages
 ###< symfony/messenger ###
+
+WEBHOOK_USERNAME=webhook
+WEBHOOK_PASSWORD=cCVMxdX8hfctDefk
```

Comment:
Same proposal about changing naming to `BASIC_AUTH_USERNAME` / `BASIC_AUTH_PASSWORD`

Reply:
Agreed and renamed